### PR TITLE
Create github action to create bundle zip file

### PR DESCRIPTION
**Pull request by silverdrake11**

### CODE REVIEW

File: .github/workflows/blank.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# This is a basic workflow to help you get started with Actions
```
Comment (abehnia) [inline .github/workflows/blank.yml]: Remove this comment.

File: .github/workflows/blank.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "scalable-stable" branch
```
Comment (abehnia) [inline .github/workflows/blank.yml]: Fix comment.

File: .github/workflows/blank.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v4
+
+      # Runs a single command using the runners shell
+      - name: Run a one-line script
```
Comment (abehnia) [inline .github/workflows/blank.yml]: Remove `Hello World` step.

File: .github/workflows/blank.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        run: echo Hello, world!
+
+      - name: Install Charmcraft
```
Comment (abehnia) [inline .github/workflows/blank.yml:25]: Specific version of Charmcraft.

File: .github/workflows/blank.yml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+        run: sudo snap install charmcraft --classic
+
+      - name: Pack with Charmcraft
+        run: charmcraft pack
+
+      - name: Upload Artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: landscape-scalable
```
Comment (abehnia) [inline .github/workflows/blank.yml:34]: Have to associate to the proper branch.

File: .github/workflows/blank.yml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+          path: ./*.zip
```
Comment (abehnia) [inline .github/workflows/blank.yml:35]: Check this.

File: .github/workflows/blank.yml [hunk 7]
```diff
@@ -0,0 +1,42 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "scalable-stable" branch
+  push:
+    branches: [ "scalable-stable", "scalable-edge" ]
```
Comment (abehnia) [inline .github/workflows/blank.yml]: Since this is branch specific, maybe include only the relevant branch.

**Comment by silverdrake11** [comment]
Ok I have addressed your comments. See workflow generated here https://github.com/silverdrake11/landscape-bundles/actions

**Review by abehnia — APPROVED** [review]

**Pull request merged**